Dockerfile

```diff
@@ -1,168 +1,81 @@
- && wget ${NETCDF4CXX_URL} \
- && tar -xzf v${NETCDF4CXX_VER}.tar.gz \
- && cd netcdf-cxx4-${NETCDF4CXX_VER} \
- && LOG_FILE=/met/logs/netcdf-cxx4-${NETCDF4CXX_VER}_configure.log \
- && echo "Configuring netcdf-cxx4-${NETCDF4CXX_VER} and writing log file ${LOG_FILE}" \
- && ./configure > ${LOG_FILE} \
- && LOG_FILE=/met/logs/netcdf-cxx4-${NETCDF4CXX_VER}_make_install.log \
- && echo "Compiling netcdf-cxx4-${NETCDF4CXX_VER} and writing log file ${LOG_FILE}" \
- && make install > ${LOG_FILE} \
- && cd /met/external_libs \
- && rm -rf netcdf
+RUN sed -i 's/policy domain="coder" rights="none" pattern="PS/policy domain="coder" rights="read | write" pattern="PS/g' /etc/ImageMagick-6/policy.xml \
+ && sed -i 's/policy domain="coder" rights="none" pattern="EPS"/policy domain="coder" rights="read | write" pattern="EPS"/g' /etc/ImageMagick-6/policy.xml \
+ && sed -i 's/policy domain="coder" rights="none" pattern="PDF"/policy domain="coder" rights="read | write" pattern="PDF"/g' /etc/ImageMagick-6/policy.xml \
+ && sed -i 's/policy domain="coder" rights="none" pattern="XPS"/policy domain="coder" rights="read | write" pattern="XPS"/g' /etc/ImageMagick-6/policy.xml
 
 #
-# Download and install HDF4 and HDFEOS.
+# Set the working directory
 #
-RUN echo "Downloading HDF4.2r3 from ${HDF4_URL}" \
- && curl -SL ${HDF4_URL} | tar zxC /met/external_libs \
- && cd /met/external_libs/HDF4.2r3 \
- && LOG_FILE=/met/logs/HDF4.2r3_configure.log \
- && echo "Configuring HDF4.2r3 and writing log file ${LOG_FILE}" \
- && ./configure --prefix=/usr/local/hdf --disable-netcdf > ${LOG_FILE} \
- && cat mfhdf/hdiff/Makefile | sed 's/LIBS = -ljpeg -lz/LIBS = -ljpeg -lz -lm/g' > Makefile_NEW \
- && mv -f Makefile_NEW mfhdf/hdiff/Makefile \
- && LOG_FILE=/met/logs/HDF4.2r3_make_install.log \
- && echo "Compiling HDF4.2r3 and writing log file ${LOG_FILE}" \
- && make install > ${LOG_FILE} \
- && echo "Downloading hdfeos from ${HDFEOS_URL}" \
- && curl -SL ${HDFEOS_URL} | tar zxC /met/external_libs \
- && cd /met/external_libs/hdfeos \
- && LOG_FILE=/met/logs/hdfeos_configure.log \
- && echo "Configuring hdfeos and writing log file ${LOG_FILE}" \
- && ./configure --prefix=/usr/local/hdfeos --with-hdf4=/usr/local/hdf CC=/usr/local/hdf/bin/h4cc > ${LOG_FILE} \
- && LOG_FILE=/met/logs/hdfeos_make_install.log \
- && echo "Compiling hdfeos and writing log file ${LOG_FILE}" \
- && make install > ${LOG_FILE} \
- && mkdir /usr/local/hdfeos/include \
- && cp include/*.h /usr/local/hdfeos/include/. \
- && cd /met/external_libs \
- && rm -rf HDF4.2r3 hdfeos
+WORKDIR /met
 
-RUN echo "Downloading GhostScript fonts from ${GSFONT_URL} into /usr/local/share/met" \
- && mkdir -p /usr/local/share/met \
- && curl -SL ${GSFONT_URL} | tar zxC /usr/local/share/met
+RUN wget https://dtcenter.ucar.edu/dfiles/code/METplus/MET/installation/tar_files.tgz \
+ && wget https://raw.githubusercontent.com/dtcenter/MET/feature_2669_proj/internal/scripts/installation/compile_MET_all.sh \
```

Comment:
We will need to remember to change this path to no longer reference the feature branch once dtcenter/MET#2672 is merged.